github/comments.go

```diff
@@ -36,6 +36,12 @@ Ammending updates the existing PR. You **DO NOT** need to open a new one.
 	return g.addUniqueComment(repo, strconv.Itoa(pr.Number), comment, "sign your commits", content)
 }
 
+func (g GitHub) addExecdriverDeprecationComment(repo octokat.Repo, pr *PullRequest, content *PullRequestContent) error {
+	comment := `Please note that concept of execdrivers is being replaced with OCI compliant binaries executed through containerd. There is an ongoing effort for switching to containerd in #20662 . Please consider porting the changes in your PR to this branch instead.`
+
+	return g.addUniqueComment(repo, strconv.Itoa(pr.Number), comment, "execdriver deprecation", content)
```

Comment:
you need to change the "execdriver deprecation" string to something unique that's part of the comment you're adding, as it is used to check if the comment is already added https://github.com/docker/leeroy/blob/96bf4b85ae2ea0d391bdf0d87de51d6ad85eb12c/github/pull_request.go#L138